test_runner/regress/test_walredo_not_left_behind_on_detach.py

```diff
@@ -0,0 +1,106 @@
+import time
+from pathlib import Path
+
+import psutil
+import pytest
+from fixtures.log_helper import log
+from fixtures.neon_fixtures import NeonEnvBuilder, PageserverApiException
+from fixtures.types import TenantId
+
+
+def assert_child_processes(pageserver_pid, wal_redo_present=False, defunct_present=False):
+    # XXX --ppid is probably GNU-only
+    children = list(psutil.Process(pageserver_pid).children())
```

Comment:
I think it returns a list of processes by default

```suggestion
    children = psutil.Process(pageserver_pid).children()
```